### PR TITLE
Update MMS integration docs 

### PR DESCRIPTION
**Pull request by vineelpratap**
# What does this PR do?

Current MMS documentation is only focused on ASR. Update the doc to show examples for TTS, LID. 

cc. @patrickvonplaten @sanchit-gandhi

### CODE REVIEW

**Review by patrickvonplaten — APPROVED** [review]
Thanks a lot for adding this!

File: docs/source/en/model_doc/mms.mdx [hunk 1]
```diff
@@ -30,21 +30,24 @@ for the same number of languages, as well as a language identification model for
 Experiments show that our multilingual speech recognition model more than halves the word error rate of 
 Whisper on 54 languages of the FLEURS benchmark while being trained on a small fraction of the labeled data.*
 
+Here are the different models open sourced in the MMS project. The models and code are originally released [here](https://github.com/facebookresearch/fairseq/tree/main/examples/mms). We have add them to the `transformers` framework, making them easier to use.
```
Comment (sanchit-gandhi) [inline docs/source/en/model_doc/mms.mdx:33]: We mention the original repository below, so not strictly necessary to re-reference it here!

File: docs/source/en/model_doc/mms.mdx [hunk 2]
```diff
@@ -30,21 +30,24 @@ for the same number of languages, as well as a language identification model for
...
   [`Wav2Vec2CTCTokenizer`].
-- MMS can load different language adapter weights for different languages via [`~Wav2Vec2PreTrainedModel.load_adapter`]. Language adapters only consists of roughly 2 million parameters 
+- You can load different language adapter weights for different languages via [`~Wav2Vec2PreTrainedModel.load_adapter`]. Language adapters only consists of roughly 2 million parameters 
   and can therefore be efficiently loaded on the fly when needed.
 
-Relevant checkpoints can be found under https://huggingface.co/models?other=mms.
-
-MMS's architecture is based on the Wav2Vec2 model, so one can refer to [Wav2Vec2's documentation page](wav2vec2).
 
 The original code can be found [here](https://github.com/facebookresearch/fairseq/tree/main/examples/mms).
```
Comment (sanchit-gandhi) [inline docs/source/en/model_doc/mms.mdx:48]: (i.e. here)

File: docs/source/en/model_doc/mms.mdx [hunk 3]
```diff
@@ -156,3 +159,81 @@ processor.tokenizer.vocab.keys()
...
+Individual TTS models are available for each of the 1100+ languages. The models and inference documentation can be found [here](https://huggingface.co/facebook/mms-tts).
+
+### **Language Identification (LID)**
+
+Different LID models are available based on the number of languages they can recognize - [126](https://huggingface.co/facebook/mms-lid-126), [256](https://huggingface.co/facebook/mms-lid-256), [512](https://huggingface.co/facebook/mms-lid-512), [1024](https://huggingface.co/facebook/mms-lid-1024), [2048](https://huggingface.co/facebook/mms-lid-2048), [4017](https://huggingface.co/facebook/mms-lid-4017). 
+
+#### Inference
+First, we install transformers and some other libraries
+```
+pip install torch accelerate torchaudio datasets
```
Comment (sanchit-gandhi) [inline docs/source/en/model_doc/mms.mdx:176]: Propagating the suggestions from https://github.com/facebookresearch/fairseq/pull/5186
```suggestion
pip install torch datasets[audio]
```

**Review by sanchit-gandhi — APPROVED** [review]
Very nice! Thanks @vineelpratap!

File: docs/source/en/model_doc/mms.mdx [hunk 4]
```diff
@@ -30,21 +30,24 @@ for the same number of languages, as well as a language identification model for
 Experiments show that our multilingual speech recognition model more than halves the word error rate of 
 Whisper on 54 languages of the FLEURS benchmark while being trained on a small fraction of the labeled data.*
 
+Here are the different models open sourced in the MMS project. The models and code are originally released [here](https://github.com/facebookresearch/fairseq/tree/main/examples/mms). We have add them to the `transformers` framework, making them easier to use.
+
+### **Automatic Speech Recognition (ASR)**
```
Comment (sgugger) [inline docs/source/en/model_doc/mms.mdx:35]: No need to add bold to a header (here and all others below)

**Review by sgugger — APPROVED** [review]
Thanks for adding those!

File: docs/source/en/model_doc/mms.mdx (hunk 1 above)
Comment (vineelpratap) [reply docs/source/en/model_doc/mms.mdx:33]: I removed the other reference.

File: docs/source/en/model_doc/mms.mdx (hunk 3 above)
Comment (vineelpratap) [reply docs/source/en/model_doc/mms.mdx:173]: Should we update the LID docs as well - https://huggingface.co/facebook/mms-lid-126  ?

**Comment by sanchit-gandhi** [comment]
Thanks for the doc updates @vineelpratap!

**Pull request merged**